zap/src/main/java/org/zaproxy/zap/extension/alert/AlertAPI.java

```diff
@@ -285,7 +302,7 @@ public JSON toJSON() {
             result = resultList;
 
             // 0 (RISK_INFO) -> 3 (RISK_HIGH)
-            ApiResponseList[] list = new ApiResponseList[4];
+            ApiResponseList[] list = new ApiResponseList[Alert.RISK_HIGH + 1];
```

Comment:
I'd prefer use of `Alert.MSG_RISK.length` or extract an actual constant for the number of risks.